sdk/src/wallet/storage/manager.rs

```diff
@@ -181,4 +181,18 @@ impl StorageManager {
             .set(ACCOUNTS_INDEXATION_KEY, self.account_indexes.clone())
             .await
     }
+
+    pub async fn set_default_sync_options(
+        &mut self,
+        account_index: u32,
+        sync_options: &SyncOptions,
+    ) -> crate::wallet::Result<()> {
+        let key = format!("{ACCOUNT_INDEXATION_KEY}{account_index}-{ACCOUNT_SYNC_OPTIONS}");
```

Comment:
opinions on this key?

Reply:
Why not just "{ACCOUNT_SYNC_OPTIONS}{account_index}"?

Reply:
I prefer keeping the same format as setting an account, as this is on top of account, not apart

Reply:
I'm also fine with it